readme.txt

```diff
@@ -122,6 +122,9 @@ Currently, the following add-ons are available for Event Tickets:
 
 == Changelog ==
 
+= [] TBD =
```

Comment:
Would you mind add the version here it should be `4.7.5`.

Thanks @jentheo

Reply:
Done!